lang/sv/index.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: swedish
+title: Semantisk versionshantering 2.0.0
+---
+
+Semantisk versionshantering 2.0.0
+==============================
+
+Sammanfattning
+-------
+
+Givet ett versionsnummer MAJOR.MINOR.PATCH, räkna upp:
```

Comment:
Agree that it may be more problematic to keep perfectly in sync with the English version. The upside is it works as an executive summary or tldr as the main text below is quite dense